yaptide/celery/utils/utils.py

```diff
@@ -81,7 +81,7 @@ def send_task_update(simulation_id: int, task_id: str, update_key: str, update_d
         "update_key": update_key,
         "update_dict": update_dict
     }
-    res: requests.Response = requests.Session().post(url=f"{flask_url}/tasks/update", json=dict_to_send)
+    res: requests.Response = requests.Session().post(url=f"{flask_url}/tasks", json=dict_to_send)
```

Comment:
why was this changed ?

Reply:
Because `tasks` endpoint does not exist and it is bad practise to have child endpoint when there is no parent endpoint. The problem occures in some other places currently.